NEWS.md

```diff
@@ -1,5 +1,7 @@
 # butcher (development version)
 
+* Added butcher methods for `clustMixType::kproto()` (@galen-ft, #235).
```

Comment:
Hope it is not a problem that I added a line to the NEWS.md. I only realised this is not on the list of things to do in the [adding-models-to-butcher.Rmd](https://github.com/tidymodels/butcher/blob/c4722313952ea2bf58068939a7b7bfb912b61736/vignettes/adding-models-to-butcher.Rmd#L114) vignette after I committed.

Reply:
Not at all! 🙏